### ISSUE TITLE
ETH Pool

### ISSUE THREAD

**Issue by DuyLvzy123** [UnamSanctam/SilentCryptoMiner#828]
Can I use pool eth?
Do you need a gpu to work?
I really confused, most of my miners still show 0 hashrate.

**Comment by UnamSanctam** [comment]
>Can I use pool eth?

Which pool?

>Do you need a gpu to work?

Yes you need a GPU for GPU mining.

>I really confused, most of my miners still show 0 hashrate.

Old version or new version? XMR miner or ETH miner? And you can change the algorithm to a less resource using one if you do not get a hashrate since it could be due to the computer specifications.

**Comment by DuyLvzy123** [comment]
> > Can I use pool eth?
> 
> Which pool?
>
> 
> > Do you need a gpu to work?
> 
> Yes you need a GPU for GPU mining.
> 
> > I really confused, most of my miners still show 0 hashrate.
> 
> Old version or new version? XMR miner or ETH miner? And you can change the algorithm to a less resource using one if you do not get a hashrate since it could be due to the computer specifications.



> > Can I use pool eth?
> 
> Which pool?
> 
> > Do you need a gpu to work?
> 
> Yes you need a GPU for GPU mining.
> 
> > I really confused, most of my miners still show 0 hashrate.
> 
> Old version or new version? XMR miner or ETH miner? And you can change the algorithm to a less resource using one if you do not get a hashrate since it could be due to the computer specifications.

Etherium not classic

**Comment by DuyLvzy123** [comment]
> > Can I use pool eth?
> 
> Which pool?
> 
> > Do you need a gpu to work?
> 
> Yes you need a GPU for GPU mining.
> 
> > I really confused, most of my miners still show 0 hashrate.
> 
> Old version or new version? XMR miner or ETH miner? And you can change the algorithm to a less resource using one if you do not get a hashrate since it could be due to the computer specifications.

2miners

**Comment by UnamSanctam** [comment]
Yes you can use 2miners, if you're mining ETH then your GPUs will need at least 4.9GB of free VRAM to be able to mine https://minerstat.com/dag-size-calculator.

**Comment by DuyLvzy123** [comment]
> Yes you can use 2miners, if you're mining ETH then your GPUs will need at least 4.9GB of free VRAM to be able to mine https://minerstat.com/dag-size-calculator.
in v2.4.1 I have this problem again "Watch dog running : No", "Miner(s): (empty)"

**Comment by UnamSanctam** [comment]
Make sure you have "Run miner after install" enabled, make sure that you started the correct miner and make sure that you're running the correct checker.

**Comment by DuyLvzy123** [comment]
> Make sure you have "Run miner after install" enabled, make sure that you started the correct miner and make sure that you're running the correct checker.

In the old version, I used ConfuserEX to obfuscate the miner virus quite easily, but the latest version has to obfuscate 2 times (watch dog, miner virus), maybe this is the cause. Can you show me how to obfuscate virus miner in current version?

**Comment by UnamSanctam** [comment]
There is no difference between the old version and the new version in that regard, but make sure you do not have the "Encrypt Resources" obfuscation enabled in ConfuserEx.

**Comment by DuyLvzy123** [comment]
> 



> There is no difference between the old version and the new version in that regard, but make sure you do not have the "Encrypt Resources" obfuscation enabled in ConfuserEx.

is it ok if i don't obfuscate?
